Makefile

```diff
@@ -25,7 +25,18 @@ anvil-start:
 	@echo "Starting Anvil..."
 	anvil --load-state contracts/scripts/anvil/state/alignedlayer-deployed-anvil-state.json 
 
-bindings: 
+# TODO: Allow enviroment variables / different configuration files
```

Comment:
We should make an issue if something should be added, to do gets lost

Reply:
Created issue https://github.com/yetanotherco/aligned_layer/issues/17